math.random

```diff
@@ -0,0 +1 @@
+put directions here
```

Comment:
Generate a random whole number between 0 and 9 and assign the value to randomNum using the Math.random and Math.floor methods.

Reply:
3

Whole number is between 1 and 10

    Math.random() generates number between 0 and 1 (with many decimal places).

etc: math.random() [randomly return : 0.19157057767733932]
(this number will still have many decimal places)

To get a random whole number, you need to multiply random generated number by 10.

etc
math.random()*10 =[randomly return : 2.9757621488533914]

    Since the number will still have many decimal places, use floor() method to round a number down/nearest integer, this will give u a value between 0 and 9. Then, u can add 1 to make it a number between 1 and 10.

etc:
math.floor(0.6)   [return 0 ]
math.floor(0.6)+ 1   [return 1]

Reply:
let num = math.random() *10;
let randomNum = math.floor(num);
console.log(randomNum);